mappings/net/minecraft/client/gui/screen/world/WorldListWidget.mapping

```diff
@@ -32,6 +71,8 @@ CLASS net/minecraft/unmapped/C_wwgrvynj net/minecraft/client/gui/screen/world/Wo
 			ARG 2 levelList
 			ARG 3 level
 		METHOD m_aehstgkl recreate ()V
+		METHOD m_fijclnvs (Lnet/minecraft/unmapped/C_tljyhzqp;Ljava/nio/file/Path;Z)V
+			ARG 3 yes
```

Comment:
"yes"? What is this?

Reply:
i dont even remember

Reply:
its the button on a confirmscreen, ill change it to `isAcceptButton`